doc/man7/provider-base.pod

```diff
@@ -90,8 +90,8 @@ provider-base
  void provider_unquery_operation(void *provctx, int operation_id,
                                  const OSSL_ALGORITHM *algs);
  const OSSL_ITEM *provider_get_reason_strings(void *provctx);
- int provider_get_capabilities(void *provctx, const char *capability,
-                               OSSL_CALLBACK *cb, void *arg);
+ int ossl_prov_get_capabilities(void *provctx, const char *capability,
```

Comment:
???? This is a function pointer not a real function.  Why the change??

I suspect it should be reverted in this file.

Reply:
capabilities.c contains the real function - guess I got confused.. Renamed